content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+ - [#13304](https://dev.haiku-os.org/ticket/13304) Auto-raised deskbar will now also auto-unraise
+
+**DriveSetup**
+ - [#10098](https://dev.haiku-os.org/ticket/10098) Encrypted disks will actually display that they are encrypted instead of empty
+
+**HaikuDepot**
+ - [#10358](https://dev.haiku-os.org/ticket/10358) When a package is queued for download and installation, it will now have the status 'queued' in the interface.
+ - [#11652](https://dev.haiku-os.org/ticket/11652) When showing all packages, the installed packages are part of the list by default. Before they were hidden by default. 
+ - [#12198](https://dev.haiku-os.org/ticket/12198) HaikuDepot can now display multiple screenshots, when the package has multiple to show.
+ - [#12428](https://dev.haiku-os.org/ticket/12428) It is possible to switch between a view with all tabs, and a view with featured packages, by using the tabs at the top of the window.
+ - [#13806](https://dev.haiku-os.org/ticket/13806) The application will now clearly show when it is downloading screenshots for the packages you are looking at.
+ - [#13808](https://dev.haiku-os.org/ticket/13808) Overall, it is more clear when HaikuDepot is contacting the web to retrieve information or updates.
+
+**Installer**
+ - [#14310](https://dev.haiku-os.org/ticket/14310) The installer now supports the user showing and selecting optional packages, when they are available on the installation medium
+
+**Launchbox**
+ - [#11237](https://dev.haiku-os.org/ticket/11237) Launchbox will now start automatically at boot, when that has been configured in its settings.
+ - [#13835](https://dev.haiku-os.org/ticket/13835) Icons of 96x96 and 128x128 pixels are now supported.
+
+**MediaPlayer**
+ - [#6628](https://dev.haiku-os.org/ticket/6628) When MediaPlayer is opened by opening a media file from Tracker, there will be a mark at the timestamp at the point where the user last stopped listening/viewing that file. 
+
+**People**
+ - [#9397](https://dev.haiku-os.org/ticket/9397) It is possible to quickly create a new Person file from Tracker, with the 'Create new...' feature
+
+**SoftwareUpdate**
+ - [#14002](https://dev.haiku-os.org/ticket/14002) When updating the system packages itself, SoftwareUpdater will now display a 'Restart' button to inform the user that a restart is necessary.
+
+**Terminal**
+ - [#15294](https://dev.haiku-os.org/ticket/15294) Support for an emulated Meta key
+
+**Tracker**
+ - [#1264](https://dev.haiku-os.org/ticket/1264) multi-range selection in Tracker and list views
+ - [#3011](https://dev.haiku-os.org/ticket/3011) Directories with a large number of files are now loaded quicker
+ - [#5126](https://dev.haiku-os.org/ticket/5126) Improvements to the Tracker API
+ - [#10529](https://dev.haiku-os.org/ticket/10529) For files that are part of a package, show information about that package in the info window.
```

Comment:
The original description of the ticket is quite restrictive: this has grown into a complete rewrite of the "File information" window and it can display all extended attributes, not just package specific information.